Tevling/Pages/Error.cshtml.cs

```diff
@@ -2,7 +2,7 @@
 using Microsoft.AspNetCore.Mvc.RazorPages;
 using Activity = System.Diagnostics.Activity;
 
-namespace Tevling.Pages;
+namespace Tevling.Pages.Error;
```

Comment:
Not sure why this change was made, but every class inside the `Pages` folder should preferably use the namespace `Tevling.Pages`.